### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by constantinpape** [platybrowser/mmpb-backend#1]
We should think about adding a license to this.
This might be a bit tricky, because this repository contains code and data,
maybe we need two separate licenses for this.

**Comment by tischi** [comment]
Yes, and I have `0.000000001` of a clue about this. Do you have more of a clue?
We could ask bio-it@embl.de

**Comment by constantinpape** [comment]
> Yes, and I have `0.000000001` of a clue about this. Do you have more of a clue?
> We could ask [bio-it@embl.de](mailto:bio-it@embl.de)

No, I also don't have a real clue what to do here. Asking bio-it is a good idea. I will write a mail later.
